genome_assembly/genome_assembly.snakefile

```diff
@@ -327,15 +330,82 @@ rule filter_contigs:
         python {params.filter_script} {input} {params.min_length} {output}
         """
 
+rule download_db:
+    """
+    Download the kraken2 database for contamination identification
+    """
+    output:
+        db = directory(config["out_dir"] + "/all_samples/kraken-db")
+    params:
+       db_location=config["out_dir"] + '/all_samples/kraken-db',
+       db=config['db'],
+       queue=config['queue'],
+       priority=config['priority'],
+       logs_dir=LOGS_DIR
+
+    shell:
+        '''
+        mkdir -p {params.db_location}
+        wget -O db.tar.gz {params.db}
+        tar -xvzf db.tar.gz -C {params.db_location}
+        rm db.tar.gz
+        '''
+
+rule run_kraken:
+    """
+    Classify organisms using Kraken2
+    """
+    input:
+        db=config["out_dir"] + '/all_samples/kraken-db',
+        assemblies=config["out_dir"] + "/per_sample/{sample}/assembly_{ena_ref}/contigs_filter.fasta"
+    output:
+        classification=config["out_dir"] + "/per_sample/{sample}/assembly_{ena_ref}/classification",
+        report=config["out_dir"] + "/per_sample/{sample}/assembly_{ena_ref}/report",
+        summary=config["out_dir"] + "/per_sample/{sample}/assembly_{ena_ref}/summary"
```

Comment:
would be better to call these `contamination.report` and `contamination.summary`, so we know what these files are.